templates/.mise.toml.tpl

```diff
@@ -84,6 +84,13 @@ echo "Tests are running"
 ## <</Stencil::Block>>
 {{- end }}
 
+[tasks.gentable]
+description = 'Generate the README.md table of arguments'
+run = """
+#!/usr/bin/env bash
+node scripts/yamltotable.js
+"""
```

Comment:
Don't need shebang if you're not executing it as a file, otherwise it has no effect

```suggestion
run = "node scripts/yamltotable.js"
```